### PR TITLE
Add failure action creator to sync channels

### PR DESCRIPTION
**Pull request by BrunoGodefroy**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -2,12 +2,14 @@ import { cancelled, put, take } from 'redux-saga/effects'
 
 export const noop = x => x
 
-export function * syncChannel (channel, successActionCreator, transform = noop) {
+export function * syncChannel (channel, successActionCreator, transform = noop, failureActionCreator = null) {
```
Comment (n6g7) [inline src/utils.js]: I'm not sure about the order of the arguments here. Do you think `transform` is more useful than `failureActionCreator`?
Maybe this should be an object with three keys to make things simpler.

**Comment by BrunoGodefroy** [comment]
@n6g7 I have refactored the arguments into an `options` object. If you like it I will update the docs accordingly.

**Comment by n6g7** [comment]
@BrunoGodefroy Amazing, that looks really good!
Let me know when you've updated the docs and I'll merge. :)

**Comment by BrunoGodefroy** [comment]
@n6g7 I have updated the docs now!

**Pull request merged**